### ISSUE TITLE
Feature request: GPU affinity (WGL_NV_gpu_affinity)

### ISSUE THREAD

**Issue by cxw42** [Gargaj/Bonzomatic#79]
I am helping organize a Shader Showdown, and we may need to render on the dual-NVidia compo machine.  If so, we will run two Bonzomatic processes concurrently, and drive them programmatically.  Would you be willing to add [WGL_NV_gpu_affinity](https://www.khronos.org/registry/OpenGL/extensions/NV/WGL_NV_gpu_affinity.txt) support so we could be sure each process was only using one of the GPUs?  I found [an implemented example](https://github.com/Eyescale/Equalizer/blob/master/tools/affinityCheck/affinityCheck.cpp) of querying, and a more complex example [here](https://github.com/OpenGLInsights/OpenGLInsightsCode/tree/master/Chapter%2027%20Multi-GPU%20Rendering%20on%20NVIDIA%20Quadro) that includes rendering.  Thanks for considering this request!

**Comment by Gargaj** [comment]
Wouldn't it be easier if you'd just fork it for yourself first and do a test run?

**Comment by cxw42** [comment]
Yep, already forked it :) .  But I have never worked with this codebase, so I thought I'd ask in case it was something easy for developers who know what they're doing :D .

**Comment by Gargaj** [comment]
It's OpenGL, noone _really_ knows what they're doing :)

**Comment by cxw42** [comment]
OK - I have an initial Nvidia version.  Would someone with two NV GPUs please try [the fork](https://github.com/cxw42/Bonzomatic/tree/gpu-affinity) and let me know if it works?  My 1050 Ti doesn't support `WGL_NV_gpu_affinity`, and I think that might be because it's the only card in the system.  (Other thoughts welcome!)

**Comment by cxw42** [comment]
Well, Demosplash's dual Titan X also doesn't support affinity. :/  Anybody have a couple of Quadros lying around? :)
